generators/app/index.babel.js

```diff
@@ -94,7 +94,7 @@ export default Base.extend({
       this.getPackageVersions(
         'deps',
         [
-          'babel-core',
+          ['babel-core', '5.0.33']
```

Comment:
You're missing a comma here. The version should also probably be 5.8.33. Lines 118/119 below need versions 5.8.33 and 5.3.3 respectively, and index.js should be rebuilt by running `npm run build`.